packages/contractkit/src/contract-cache.ts

```diff
@@ -119,18 +123,21 @@ export class WrapperCache {
   getGovernance() {
     return this.getContract(CeloContract.Governance)
   }
+  getGovernanceApproverMultiSig() {
+    return this.getContract(CeloContract.GovernanceApproverMultiSig)
```

Comment:
Are you sure this will work? I think these getters look these contracts up from the registry, and these contracts probably shouldn't be registered as they're not part of the "core" suite